### PR TITLE
Turing Initial allocation & Vesting

### PR DESCRIPTION
**Pull request by irsal**

### CODE REVIEW

File: distribution/turing_vesting.json [hunk 1]
```diff
@@ -0,0 +1,303 @@
+[
+  [
+    1651777200,
+    [
+      ["6AWgGe7oZuvc8542mBGwJVs1FVZoQdNH68bsaNvhzSY2xjYN", 2000000000000000000000000000],
```
Comment (irsal) [inline distribution/turing_vesting.json:5]: Needs to be changed

File: distribution/turing_vesting.json [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+      ["66fiEgBVzAGs75UEqwGyXnJBbvujb9NwbvrXbvoqWYbK6Uui", 41666666666666666],
+      ["67grpvYywCTJ5gCNKfDMv4WMc416yUc43DMV7Xt31rUFHexc", 27777777777777777]
+    ] 
+  ],
+  [
+    1654455600,
+    [
+      ["6AWgGe7oZuvc8542mBGwJVs1FVZoQdNH68bsaNvhzSY2xjYN", 200000000000000000],
+      ["68UQrHt7FQ171AF5nvkUV2jChxhJv58tp8xXZ6spzMnvWvFN", 41666666666666666],
+      ["66fiEgBVzAGs75UEqwGyXnJBbvujb9NwbvrXbvoqWYbK6Uui", 416666666666666666666666666],
```
Comment (irsal) [inline distribution/turing_vesting.json:16]: needs to be changed

File: distribution/turing_vesting.json [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+  [
+    1725562800,
+    [
+      ["68UQrHt7FQ171AF5nvkUV2jChxhJv58tp8xXZ6spzMnvWvFN", 41666666666666666],
+      ["66fiEgBVzAGs75UEqwGyXnJBbvujb9NwbvrXbvoqWYbK6Uui", 41666666666666666],
+      ["67grpvYywCTJ5gCNKfDMv4WMc416yUc43DMV7Xt31rUFHexc", 27777777777777777]
+    ]
+  ],
+  [
+    1728154800,
```
[the remark this answers is not in the document]
Comment (irsal) [reply distribution/turing_vesting.json:254]: `1730836800` is nov was this a stale comment?

[the remark this answers is not in the document]
Comment (chrisli30) [reply distribution/turing_vesting.json:254]: Verified that the row for Nov is added, 1730836800, November 5, 2024 8:00:00 PM

**Comment by irsal** [comment]
Closing this PR. Will do 2 PRs instead:
1. cleaned up json files
2. for the unit tests

**Pull request closed without merging**